genutil/common.go

```diff
@@ -336,17 +360,21 @@ func FindAllChildren(e *yang.Entry, compBehaviour CompressBehaviour) (map[string
 				// expect that there are some duplicates here for applied configuration leaves
 				// (those that appear both in the "config" and "state" container).
 				if e.Dir[currChild].Name == deprioData {
-					// Compress out (do not map) choice/case nodes that are in the
-					// config or state container. This is again specifically for the
-					// OpenConfig routing policy model.
-					// Further, if the name is a duplicate to one that's already in the
-					// prioritized container, we must drop the entry, and ignore any error
-					// that is returned, as we allow those duplicates.
+					ch := map[string]*yang.Entry{"": configStateChild}
 					if util.IsChoiceOrCase(configStateChild) {
-						// Duplicates could occur in a choice/case as well.
-						errs = addNonChoiceChildrenDuplist(directChildren, configStateChild, prioNames, errs)
-					} else if !prioNames[configStateChild.Name] {
-						errs = addNewChild(directChildren, configStateChild.Name, configStateChild, errs)
+						// Compress out (do not map) choice/case nodes that are in the
+						// config or state container. This is again specifically for the
+						// OpenConfig routing policy model.
```

Comment:
Suggest to remove this comment re: the OC routing policy model, since this has been removed.